src/store/getters.ts

```diff
@@ -206,5 +206,11 @@ export default {
   /** The parties list from the API. */
   getParties (state: StateIF): Array<PartyIF> {
     return state.parties
+  },
+
+  /** The entity busness address. */
+  gettBusinessAddress (state: StateIF): OfficeAddressIF {
+    return state.businessAddress
   }
```

Comment:
Now that we have a nice getter for this, could you please update FirmsAddressList.vue:122 to replace the State with Getter?

Reply:
TY